Px.Search.Lucene/LuceneAnalyzer.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Lucene.Net.Analysis.En;
+using Lucene.Net.Analysis.Sv;
+using Lucene.Net.Analysis.No;
+using Lucene.Net.Analysis.Standard;
+using Lucene.Net.Analysis;
+using Lucene.Net.Util;
+
+namespace Px.Search.Lucene
+{
+    public class LuceneAnalyzer
+    {
+        internal static LuceneVersion luceneVersion = LuceneVersion.LUCENE_48;
+
+        internal static Analyzer GetAnalyzer(string language, bool useStandardAnalyzer)
```

Comment:
Should we add more language specific analyzer for languages that have them and that we how there are existing users e.g.:
- Finish (Fi)
- Greek (El)
- Basque (Eu)
- Spanish (Es)
- French (Fr)
- Arabic (Ar)
- Italian (It)
- Latvian (Lv)
- Indonesian (In)
- Armenian (Hy)
- German (De)
- Danish (Da)

Reply:
Probably easier to do it now  :-)  What to do with "unlisted" languages in the long language-if : warning or error?

Reply:
Yes, but how will we remember to add new langauges. Well, if nobody complains then it is ok :-)

Reply:
We can just mention in the documentation which languages are stemmed. I think people will give feedback if their language is missing :-)